components/policy/resources/templates/policy_definitions/BraveSoftware/BraveVPNDisabled.yaml

```diff
@@ -0,0 +1,38 @@
+caption: Disable Brave VPN
+default: null
+desc: |-
+  Disable Brave VPN in Brave.
+
+  Allows users to use Brave VPN to block trackers and encrypt every connection to the Web, on every app, even outside the Brave browser.
```

Comment:
cc: @mattmcalister on the wording here! I copied it from https://brave.com/firewall-vpn/. It'll show in the Windows group policy editor which looks like this:
![image](https://github.com/user-attachments/assets/0fef3fa1-3e07-401f-ab80-960ce64e02a8)